pkg/export/setup/setup.go

```diff
@@ -239,6 +243,43 @@ func ExtraArgs() ([]string, error) {
 	return shlex.Split(os.Getenv(ExtraArgsEnvvar))
 }
 
+type MetadataOpts struct {
```

Comment:
> @pintohutch  [`7fe3d0b` (#974)](https://github.com/GoogleCloudPlatform/prometheus-engine/pull/974/commits/7fe3d0b10a0a0258d55440ef27bc473102e02fe1#r1661201261)
> question: Why separate the MetadataOpts config?

My goal was to split up configurations used in setup.go and export.go. Previously these values were just variables in `FromFlags`.

I can combine `MetadataOpts` and `HAOpts` into something like `SetupOpts` if that makes more sense, but the separation meant clearer separation of concerns.